docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -0,0 +1,331 @@
+{ref}`summarization <old-summary-example>`.
+Some of the information in these references is out of date.
+```
+
+## Prover Settings
+
+In this subsection we list some option combinations that have helped preventing
+timeouts in the past. We group the options by timeout causes they are most
+relevant for. 
+
+### Dealing With a High Path Count
+
+The Certora Prover internally divides each verification condition into smaller
+subproblems and attempts to solve them separately. This technique is called
+*control flow splitting*. For a more detailed explanation of how control flow
+splitting works, see [this page](control-flow-splitting).
+
+We list a few option combinations that can help in various settings. There is
+a tradeoff between spending time in different places: The Prover can either try
+to spend much time at a low splitting level in the hope that no further
+splitting will be needed, or it can split quickly in the hope that the
+subproblems will be much easier to solve. The first variant ("lazy splitting")
+is weak when the shallow splits are too hard, and time spent on them is wasted.
+The second variant ("eager splitting") is weak when we end up with too many
+subproblems; the number of splits is worst-case exponential in the splitting
+depth.
+
+When the relevant source code is very large, the shallow splits have a chance of
+being too large for the solvers, thus eager splitting might help.
+
+```
+--prover_args "-smt_initialSplitDepth 5 -depth 15"
+```
+
+When there are very many subproblems that are of medium difficulty there is a
+chance that the Prover has to split too often (not being able to "close" any
+sub-splits). Then, a lazier splitting strategy could help. We achieve lazier
+splitting by giving the solver more time to find a solution before we split a
+problem.
+
+```
+--prover_args "-mediumTimeout 30 -depth 5"
+```
+
+It can also help to have splitting run in parallel; this goes together with
+choosing a smaller solver portfolio, so the available cores can be used for
+different splits, rather than different solvers.
+
+```
+--prover_args "-smt_splitParallel true -solvers [z3:def,cvc5:def]"
+```
+
+### Dealing With Nonlinear Arithmetic
+
+Nonlinear integer arithmetic is in general the hardest part of the formula's
+that Certora Prover is solving (being undecidable in general).
+
+#### Running with Yices
+
+A different choice of solver sometimes helps. The *Yices* SMT solver ([home
+page](https://yices.csl.sri.com/)) is not used by default since it is not
+compatible with our default hashing scheme. 
+
+The following setting sets a hashing scheme that is compatible with Yices. Since
+Yices is in the default portfolio, it is included automatically then.
+
+```
+--prover_args "-smt_hashingScheme plainInjectivity"
+```
+
+Optionally, we can further prioritize the usage of Yices by decreasing the size
+of the solver portfolio. With the `-solvers` option set as follows, Certora
```

Comment:
`, Certora Prover` -> `, The Certora Prover`?

Reply:
hm, I remember "article vs no article" rules are very complicated ... with sounds ok to me, but not certain ...  I don't think I want to capitalize the article ...